src/etl/sds/tests/changelog/conftest.py

```diff
@@ -9,6 +9,10 @@
 
 import boto3
 import pytest
+from domain.core.cpm_product.v1 import CpmProduct
```

Comment:
Anywhere you have CPMProduct and ProductTeam now need to change to EPRProduct and the EPR ProductTeam